lsp/src/import.ml

```diff
@@ -20,6 +20,13 @@ module Queue = Queue
 module String = struct
   include Stdune.String
 
+  let contains_sub s ~sub =
```

Comment:
Please don't use `Str`. It's not thread safe. Either copy some existing implementation of this or just quickly implement something yourself.